dsn-2018/bare_conf.tex

```diff
@@ -0,0 +1,1350 @@
+        blocks which hash to the same value.
+\end{itemize}
+Note that while a second preimage is necessarily a collision, it is also a
+stronger result with higher use for an attacker.
+
+From an attacker's perspective, finding a preimage or second preimage has been
+difficult: the current published bound for a preimage attack in MD5 is a cost
+of $2^{123.4}$ by Y. Sasaki and K. Aoki \cite{SasakiPreimage}. Further, while
+J. Kelsey and B. Schneier proposed a method for finding second preimages
+in less than $2^n$, we note that this requires rather long messages
+\cite{SchneierSecondPreimage}. However, collision attacks are well within reach
+of adversaries, and well past theoretical attacks. The work of
+H. Dobbertin \cite{Dobbertin1998}, X. Wang \cite{cryptoeprint:2004:199},
+M. Schl{\"a}ffer \cite{Schlaffer2006} and others demonstrate the ease with which
+collisions can be found.
+
+From a cryptographic perspective, the existence of a feasible collision attack
+breaks the requisite properties, thus deprecating the function's use. In some
+instances however, there continues to be widespread use of deprecated hash
+functions. One widespread example is the continued use by Git of SHA-1, despite
+M. Stevens et al. publishing a full SHA-1 collision in February of 2017
+\cite{cryptoeprint:2017:190}. However, this attack had a bound of $2^{63.1}$ --
+faster than the theoretical $2^{80}$ but still requiring significant compute
+resources to find.
+
+We seek to address the problem of evaluating the use cases for a collision,
+to determine whether or not a collision affects a particular system. This
+paper contributes several techniques to logical cryptanalysis and uses these
+techniques to derive metrics of the trustworthiness of hash functions. We claim
+the following results are novel in the field:
+\begin{itemize}
+    \item We define the neighborhood of a class of collisions and show that
+        it is frequently non-empty.
+    \item We discuss techniques for finding useful collisions within a
+        class of collisions.
+    \item We propose additional techniques for building new classes of
+        collisions and show that collisions in MD4 can be found inductively.
+    \item We propose five universal metrics for evaluating the utility of
+        a class of collisions.
+\end{itemize}
+We propose that collision classes of high utility are closer to a second
+preimage attack in the amount of flexibility they provide to an attacker,
+whereas classes of low utility may not affect all systems which use a
+particular hash function.
+
+\textbf{TODO - Finish Organization}
+The remainder of this paper is organized as fol
+lows. In Section
+\ref{Sec:Related} we discuss previous results and show how our research
+expands upon prior work. In Section \ref{Sec:Terminology}, we discuss the
+terminology and notations we use throughout the remainder of the paper.
+In Section \ref{Sec:Intuition}, we give the intuition between measuring
```

Comment:
Not quite sure I understand this part. "we give the intuition between measuring the utility of a collision" and what?